docs/sdk-docs/connecting/README.md

```diff
@@ -1,23 +1,106 @@
-# Connecting to a Test Network
+# Connecting
+The SDK provides out of the box support for the deployed networks on `Görli`, `Ropsten`, `Rinkeby`, and `Mainnet`.
 
-Connecting to a Raiden test network is automatically done by the Light Client SDK. The SDK will automatically connect to the proper Raiden network based on the detected network id on your web3 provider.
+If you need to use the sdk on a private network, or a custom deployment you can find more information in the following [guide](../private-chain/README.md).
 
-The SDK provides out of the box support for the deployed networks on Ropsten, Rinkeby, and Mainnet.
+## Raiden Light Client test environment
+For development purposes, the Light Client uses a standalone environment. The dApp deployment [lightclient.raiden.network](https://lightclient.raiden.network/) and the development version served by 'npm run serve' also conforms to this configuration. 
 
-If you want you can also use the Light Client SDK to connect to networks that are not officially supported.
+This environment uses:
 
-To connect for example on a privately deployed Raiden network,
-you can initialize the SDK as usual, and pass the Contract Info as the fourth parameter of the create function.
+- A specific version of [Raiden](https://github.com/raiden-network/raiden/commit/2e741dfdf4bfa564dec760abd5e3d8b2c9d30715)
```

Comment:
Pyinstaller has an issue in the tagged commit and building the binary requires these changes that fix the build applied on top. Otherwise, the binary creation will fail with an error.

I don't see why we should mention these steps here. These are only required if you want to build binaries, they don't affect anything else. The idea is that we are building the binaries to make them available for everyone that wants to use them. 

This is a one-off operation that we should not have to repeat.

Reply:
Unfortunately, the changes are really needed otherwise the build will fail.
But you could also stay with the docker image which is the easiest and proposed way forward I'd say.

Reply:
Honestly, I would do the macOS build, however, I have no access to a mac so I cannot do that. They only reason I wanted to have binaries is that it can be some times easier for people to use the binary instead of going the Docker/virtual environment way.